src/components/FormSectionHeader/FormSectionHeader.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Grid, makeStyles } from '@material-ui/core';
+
+const borderColor = '#404040';
```

Comment:
Just a note that the colors and fonts in this file should eventually be moved to a `src/styles/theme.js` file using Material-UI theming (future task). This works until then.